environment.go

```diff
@@ -299,3 +349,18 @@ func getClientConfig(baseURL string) (*ClientConfig, error) {
 
 	return c, nil
 }
+
+// copy of deepCopy in util package
```

Comment:
Is there really a cycle if you use the version in util?

Reply:
Yep, util's tests include testutil, which includes the root package.